src/Request/uapi-request.js

```diff
@@ -169,9 +174,7 @@ module.exports = function uapiRequest(
       .then(validateSOAP)
       .then((res) => {
```

Comment:
We could return to the original (if it works with provider passed like above):
```
.then(parseFunction.bind(uParser))
```

Reply:
Indeed!